.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: GitHub Actions Demo
+on: [pull_request]
```

Comment:
Up to you, but while we encourage you to play with this setting you may find it best to leave this on for pushes as well for future, so that you can get feedback when you're partway through a feature and not just when you're ready to merge.